smoke-tests/apps/Sampling/src/smokeTest/resources/applicationinsights-no-ingestion-sampling.json

```
@@ -0,0 +1,10 @@
+{
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 100,
+    "ingestionSide": false
```

It may be more explict from a customer perspective:

```suggestion
    "portailIngestion": false
```

---

i'm not sure about "portal" since portal is technical the U/X (I think)

what do you think of just `"ingestion"`?

here's the relevant docs:
https://learn.microsoft.com/en-us/azure/azure-monitor/app/sampling-classic-api#ingestion-sampling

---

I am OK with that. 

We could provide the doc link (https://learn.microsoft.com/en-us/azure/azure-monitor/app/sampling-classic-api#ingestion-sampling) in the documentation of the `ingestion` property.

---

can we stick with product's term: ingestion sampling. 
I would suggest "ingestionSamplingEnabled" default is on?

---

btw, note that I've put this directly under the stable `"sampling"` config node

another option is to put it under the preview node

```
{
  "preview": {
    "ingestionSampling": false
  }
}
```

---

i think preview is better. sampling is applicable to java agent's sampling rule.

---

> sampling is applicable to java agent's sampling rule.

I didn't understand this comment

there are three different types of sampling (fixed-rate, adaptive and ingestion) https://learn.microsoft.com/en-us/azure/azure-monitor/app/sampling-classic-api

so I think it makes sense to keep them together (unless we want to preview the config option first)

---

i meant java agent sampling rates vs. ingestion's sampling rate

---

resolving this conversation as everything has changed now, ptal 😅